bittensor/_wallet/wallet_impl.py

```diff
@@ -28,16 +28,16 @@
 from termcolor import colored
 import bittensor
 
-def display_mnemonic_msg( kepair : Keypair ):
+def display_mnemonic_msg( keypair : Keypair, key_type : str ):
     """ Displaying the mnemonic and warning message to keep mnemonic safe
     """
-    mnemonic = kepair.mnemonic
+    mnemonic = keypair.mnemonic
     mnemonic_green = colored(mnemonic, 'green')
     print (colored("\nIMPORTANT: Store this mnemonic in a secure (preferable offline place), as anyone " \
                 "who has possesion of this mnemonic can use it to regenerate the key and access your tokens. \n", "red"))
-    print ("The mnemonic to the new key is:\n\n%s\n" % mnemonic_green)
+    print ("The mnemonic to the new %s is:\n\n%s\n" % (key_type, mnemonic_green))
     print ("You can use the mnemonic to recreate the key in case it gets lost. The command to use to regenerate the key using this mnemonic is:")
-    print("bittensor-cli regen --mnemonic %s" % mnemonic)
+    print("btcli regen_%s --mnemonic %s" % (key_type, mnemonic))
     print('')
```

Comment:
Hey @ndubuisx this is actually really badly formatted python 2 style code. Can you reformat the print statements to  

```print ("The mnemonic to the new key is:\n\n{}\n".format(mnemonic_green))``` 

and update the rest accordingly?